nextflow.config

```
@@ -155,13 +156,13 @@ profiles {
     }
     docker {
         docker.enabled         = true
-        docker.userEmulation   = true
         conda.enabled          = false
         singularity.enabled    = false
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
         apptainer.enabled      = false
+        runOptions             = '-u $(id -u):$(id -g)'
```

that wont work.
```suggestion
        docker.runOptions             = '-u $(id -u):$(id -g)'
```
See https://github.com/nf-core/tools/pull/2607